### PR TITLE
Rucio demo version #5951 #5940

### PR DESCRIPTION
**Pull request by dciangot**

### CODE REVIEW

File: bin/htcondor_make_runtime.sh [hunk 1]
```diff
@@ -10,11 +10,11 @@ STARTDIR=$PWD/tmp/runtime
 CRAB3_VERSION=3.3.0-pre1
 
 WMCOREDIR=$STARTDIR/WMCore
-WMCOREVER=1.1.14.crab1
+WMCOREVER=1.2.6
```
Comment (belforte) [inline bin/htcondor_make_runtime.sh]: is the version here really relevant ? I think it is overwritten later with what's in the spec file. Otherwside we'd be in troubles already.

File: bin/htcondor_make_runtime.sh [hunk 2]
```diff
@@ -10,11 +10,11 @@ STARTDIR=$PWD/tmp/runtime
...
 WMCOREREPO=dmwm
 
 CRABSERVERDIR=$STARTDIR/CRABServer
-CRABSERVERVER=3.3.1706.rc2
+CRABSERVERVER=3.3.1909.rc1
```
Comment (belforte) [inline bin/htcondor_make_runtime.sh]: I quite confidend that the CRABSERVER version here is only a dummy placeholder. We could probably rewrite the script w/o it, but in any case better have it pointing to the 2 years old one to make it clear.

File: scripts/task_process/task_proc_wrapper.sh [hunk 3]
```diff
@@ -11,13 +11,27 @@ function cache_status {
...
-    timeout 15m python task_process/transfers.py
-    err=$?
-    if [ $err -eq 137 ] || [ $err -eq 124 ]; then
-        log "ERROR: transfers.py exited with process timeout";
-    elif [ $err -eq 0 ]; then
-        log "transfers.py exited.";
-    else log "ERROR: transfers.py exited with $err";
+
+    if [[ -f task_process/transfers.txt ]]; then
+        # TODO: automatic swtich to RUCIO transfers
```
Comment (belforte) [inline scripts/task_process/task_proc_wrapper.sh]: I do not understand this TODO

**Comment by belforte** [comment]
@dciangot 
I verified that the changes in bin/htcondor_make_runtime.sh  are indeed irrelevant since we
always use $CRAB_OVERRIDE_SOURCE.
So to keep this PR on point and avoid headached, please avoid touching that file.

Other then that it is harmless, aside from the changes to PostJob that are too much for me to review in short time. So I suggest that you roll back the useless changes above and then we
install in PreProd to see how it goes. It will be easier to review PostJob while being able to see it in action.

File: scripts/task_process/task_proc_wrapper.sh (hunk 3 above)
Comment (dciangot) [reply scripts/task_process/task_proc_wrapper.sh]: because it's solved... to be removed

File: bin/htcondor_make_runtime.sh (hunk 2 above)
Comment (dciangot) [reply bin/htcondor_make_runtime.sh]: mmm.. don't know.. without this the tarball made by hand was not working for me... And anyway, if it is not used we either keep it updated or we remove it. Am I missing something?

Comment (belforte) [reply bin/htcondor_make_runtime.sh]: maybe we should review your updateRuntime.sh. Yes this part could use some cleanup. So far it was ... it works, don't fix it.

**Comment by amaltaro** [comment]
Hi Stefano, just out of curiosity, does this PR mean that CRAB is not going to use the Rucio client APIs we have made available under WMCore/Services/Rucio?

**Comment by belforte** [comment]
I am not on top of this. Please ask @dciangot. Besides I have no idea that you
had a Rucio API in WMCore, nor what it does.

**Pull request merged**